package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "clean": "rimraf dist",
     "build:webpack": "NODE_ENV=production $(npm bin)/webpack --config webpack.config.prod.js",
-    "build": "npm run clean && npm run build:webpack",
+    "build": "npm run clean && npm cache clear && npm rebuild node-sass && npm run build:webpack",
```

Comment:
Just one thing, do you think we could move the `npm cache clear && npm rebuild node-sass` to the `clean` script? That stuff sounds like it's part of 'cleaning'